### PR TITLE
Adding version command to console

### PR DESCRIPTION
**Pull request by josefnpat**
In an attempt to get to better know the code, I have added a simple feature that will help folks who want to check the version, who are also either full screened or don't have a window manager with window titles.

Feel free to close the PR if this feature is a waste of time.

### CODE REVIEW

File: src/console.c [hunk 1]
```diff
@@ -858,6 +858,15 @@ static void onConsoleKeymapCommand(Console* console, const char* param)
 	commandDone(console);
 }
 
+static void onConsoleVersionCommand(Console* console, const char* param)
+{
+	printBack(console, "\n");
+	char msg[64] = {0};
+	sprintf(msg, "%i.%i.%i", TIC_VERSION_MAJOR, TIC_VERSION_MINOR, TIC_VERSION_PATCH);
+	consolePrint(console, msg, CONSOLE_BACK_TEXT_COLOR);
```
[the remark this answers is not in the document]
Comment (josefnpat) [reply src/console.c]: oh, nice. I'll do that asap!

[the remark this answers is not in the document]
Comment (nesbox) [reply src/console.c]: ping :)

[the remark this answers is not in the document]
Comment (josefnpat) [reply src/console.c]: Sorry about that, I had to rebuild my machine this weekend. I still have this on my todo list, and ask for a little more time, unless you want to release today <3

[the remark this answers is not in the document]
Comment (nesbox) [reply src/console.c]: np :)

**Pull request merged**